server/src/views/emails/email-reset-password.ejs

```diff
@@ -0,0 +1,32 @@
+<% /* Note: This is injected into `views/layouts/layout-email.ejs` */ %>
+<p style="margin-bottom: 25px">Dear <%= fullName %>,</p>
+<p style="margin-bottom: 25px">
+  Someone requested a password reset for your account. If this was not you, please disregard this
```

Comment:
"A password reset has been requested for your account. If this wasn't made by you, please disregard..."

i think this is a better sentence to make it sound not so sus.